examples/onprem_demo/README.md

```diff
@@ -0,0 +1,70 @@
+- Launches two virtual machines in the on-prem project: 
+    - one Linux instance acting as a forward proxy for all internet-bound requests; and 
+    - one Windows instance representing an on-prem server.  You'll use this as a jumphost to RDP into the Windows instance in the VPC SC project.
+- Configures routes and DNS resolution for [Google Private Access](https://cloud.google.com/vpc/docs/configure-private-google-access) in the VPC SC project, so GCP API calls from within your project never traverse the public internet
+- Enables VPC Service Controls.  
+
+You can optionally enable VPC SC using the Cloud Console instead of Terraform if you'd like to poke around at some of the features.
+
+The resources in this demo cost approx $14/day (or $0.58/hour) if you leave them up and running.  Fortunately, Terraform makes it easy to spin the resources up and down on-demand.
+
+## An important note about security
+If you're going to modify this code and check it in to your code repo:
+- Do not check in the service account .json file.  A bad guy who accesses this file could do serious damage to your project.
+- Do not check in a completed `terraform.tfvars` file.  A bad guy who sees your VPN public IP's and shared secrets could gain access to your network.
+
+We recommend that you delete the service account .json file and delete the service account altogether when you've completed running through this demo.
+
+## Requirements
+To complete this demo, you'll need:
+- A Google Cloud [organization](https://cloud.google.com/resource-manager/docs/quickstart-organizations) mapped to a domain that you own with at least one project already created.
+- A valid billing account where you have Billing Account User privileges.
+- A service account with Project Creator privileges.  Steps for provisioning this are included below, but those steps assume you have Organization Admin privileges.
+- Terraform v0.11.13 with provider.google v2.6.0 (the only versions I've tested; run `terraform version` to see what you've got).
+
+## Instructions
+
+0. Set up the service account needed to run the Terraform code.
+    - In an existing project, create the service account.  Copy and paste its email address (the big, long, `gserviceaccount.com` address) to a scratch pad.  Download its .json secret key file into the `onprem_project/` and also copy it into the `vpc_sc_project` folders.  Be **extremely careful** with this file; delete it when you're done with this example.
+    - Go to the [IAM admin](https://console.cloud.google.com/iam-admin/iam) section of your *organization.*  (You should see your domain name at the top of the screen, not an individual project name).  Click Add.  In the "New Members" box, paste the email address of the service account you created.  Under the Roles dialog, search for "Project Creator."  Grant that role by clicking Save.  Your service account can now create projects and act as a project owner for those projects.
```

Comment:
```suggestion
    - Go to the [IAM admin](https://console.cloud.google.com/iam-admin/iam) section of your *organization.*  (You should see your domain name at the top of the screen, not an individual project name).  Click Add.  In the "New Members" box, paste the email address of the service account you created.  Under the Roles dialog, search for "Project Creator."  Grant that role by clicking Save.  Your service account can now create projects and act as a project owner for those projects.
    -  Add the additional Organization level permissions to the service account mentioned [here](https://github.com/terraform-google-modules/terraform-google-vpc-service-controls#organization-level-permissions) to  make sure your service account can create VPC SC otherwise known as Access Context Manager resources.
```

Reply:
Done.